source/admin_manual/list_of_events.rst

```diff
@@ -972,6 +974,32 @@ POP3
    :field remote_port @added;v2.4.0,v3.0.0: POP3 connection's remote (client) port.
 
 
+.. dovecot_event:field_group:: pop3_command
+   :inherit: pop3_client
+
+   :field cmd_name @added;v3.0.0:
```

Comment:
v2.4.0 also here and elsewhere. Except you're now marking the fields as being added. It should be the whole pop3_command being added. I think `:added: v2.4.0;v3.0.0`